source/sass/buyers-guide/views/product.scss

```diff
@@ -107,20 +107,7 @@
 }
 
 // Sad Cases
-.criterion-uses_encryption.criterion-false,
-.criterion-privacy_policy_reading_level.criterion-13,
-.criterion-privacy_policy_reading_level.criterion-14,
-.criterion-privacy_policy_reading_level.criterion-15,
-.criterion-privacy_policy_reading_level.criterion-16,
-.criterion-privacy_policy_reading_level.criterion-17,
-.criterion-privacy_policy_reading_level.criterion-18,
-.criterion-privacy_policy_reading_level.criterion-19,
-.criterion-share_data.criterion-true,
-.criterion-must_change_default_password.criterion-false,
-.criterion-security_updates.criterion-false,
-.criterion-delete_data.criterion-false,
-.criterion-child_rules.criterion-false,
-.criterion-manage_security.criterion-false {
+.criterion-bad {
```

Comment:
Looks like the mapping of "reading levels >=13 are bad" didn't get translated into the refactor.